agenta-cli/agenta/cli/variant_commands.py

```diff
@@ -9,21 +9,24 @@
 import questionary
 import toml
 from agenta.cli import helper
-from agenta.client import client
 from agenta.cli.telemetry import event_track
 from agenta.client.api_models import AppVariant, Image
 from agenta.docker.docker_utils import build_tar_docker_container
 
+from agenta.client.client import ClientWrapper
+
+client_wrapper = ClientWrapper()
```

Comment:
At the point of time of calling this, the config.toml does not exist. This won't work